plugins/ruby-foreman-opennebula/debian/control

```diff
@@ -0,0 +1,13 @@
+Source: ruby-foreman-opennebula
+Section: ruby
+Priority: extra
+Maintainer: IaaS team IVI.RU <ops@ivi.ru>
+Build-Depends: debhelper (>= 9), foreman-nulldb (>= 3.7), foreman-assets (>= 3.7), ruby-foreman-deface
```

Comment:
You don't use deface
```suggestion
Build-Depends: debhelper (>= 9), foreman-nulldb (>= 3.7), foreman-assets (>= 3.7)
```

Reply:
Yes. It was added as dependency because of the polyglot gem module. Now I included the previous version of polyglot gem to the package to avoid conflict with deface deb and removed this dependency.